app/javascript/mastodon/components/status_content.js

```diff
@@ -46,10 +51,15 @@ export default class StatusContent extends React.PureComponent {
     }
   }
 
+  componentDidMount () {
+    this._updateStatusLinks();
+  }
+
   componentDidUpdate () {
     if (this.props.onHeightUpdate) {
       this.props.onHeightUpdate();
     }
+    this._updateStatusLinks();
```

Comment:
They are applied by mistake then, because they don't do anything - permalink is already a react component with its own onClick handler. So removing that is fine.